assets/js/user.js

```diff
@@ -14,10 +14,17 @@ const updateUsername = function () {
     } else {
         console.log("The session has expired or the user is not logged in.");
         // Hide logout and profile dropdown items when user is not logged in
-        document.getElementById("logout-button-item").classList.add("d-none");
-        document.getElementById("profile").classList.add("d-none");
-        // Display login dropdown item
-        document.getElementById("login-button-item").classList.remove("d-none");
+        const logout_button_item = document.getElementById("logout-button-item");
```

Comment:
@sananand007 This part of logic changes could cause "log out" button under guest user. 
You may want to roll it back and check if it works.

Reply:
ahh .. may be this is causing the problems .. good catch